src/fu-engine-emulator.h

```diff
@@ -12,6 +12,9 @@
 #define FU_TYPE_ENGINE_EMULATOR (fu_engine_emulator_get_type())
 G_DECLARE_FINAL_TYPE(FuEngineEmulator, fu_engine_emulator, FU, ENGINE_EMULATOR, GObject)
 
+/* the maximum times a device can use FWUPD_DEVICE_FLAG_ANOTHER_WRITE_REQUIRED */
+#define FU_ENGINE_EMULATOR_WRITE_COUNT_MAX 5
```

Comment:
why does this need to be limited?  It makes me wonder if we're going to have problems with emulating any specific devices that use a ton of another write required.

Reply:
although granted if it is ever a problem then we can just bump it